### PR TITLE
Updating ops-manager client

### PR DESCRIPTION
**Pull request by colm-quinn**
## Proposed changes
Updating ops-amanger go client to latest version, which contains fix for dropping tls password on config updates

_Jira ticket:_ https://jira.mongodb.org/browse/CLOUDP-92485

## Get automation config, now containing missing fields:
`./mongocli ops-manager automation get`
```
  "tls": {
    "autoPEMKeyFilePath": "<example>",
    "autoPEMKeyFilePwd": "<example>",
    "CAFilePath": "<example>",
    "clientCertificateMode": "OPTIONAL"
  },
  "version": 48
}
```

## Result of running showdown command, which sends a new automation config:
```
./mongocli ops-manager clusters shutdown myReplicaSet
? Are you sure you want to shutdown: myReplicaSet Yes
```
image

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -246,6 +246,8 @@ go.mongodb.org/atlas v0.8.1-0.20210617105011-6061b0945dd4 h1:vLw72G0G30fDpTP6rsz
 go.mongodb.org/atlas v0.8.1-0.20210617105011-6061b0945dd4/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
 go.mongodb.org/ops-manager v0.22.1-0.20210617143723-fa7bb89af282 h1:iQjUqcG8DO/541s46U9FpxhE0bGTMypjaUy1bAn7vM0=
 go.mongodb.org/ops-manager v0.22.1-0.20210617143723-fa7bb89af282/go.mod h1:13raq36D7hiSiDbnBGB+MChiRF3HKd9wZPkZHkZnjR8=
+go.mongodb.org/ops-manager v0.22.1-0.20210618124154-28a24c0e3fb0 h1:bT5Mvz0zNJOrsj+G6e9X9V50Wjcy8afj+rges6gm9UQ=
```
Comment (fmenezes) [inline go.sum:249]: for some reason it did not delete the previous two lines, can you run `go mod tidy`?

Comment (threebee) [reply go.sum:249]: TIL

Comment (colm-quinn) [reply go.sum:247]: Good spot! Done

**Review by fmenezes — APPROVED** [review]

**Comment by colm-quinn** [comment]
Test failures also on master.

**Pull request merged**